### PR TITLE
[syncd] Fix bulk multi attrs for same key db update

### PR DESCRIPTION
**Pull request by kcudnik**
attributes will be set for the same key, then when we want to, push them to redis database, we need to combine all attributes, to a single vector of attributes

issue was related to bulk set operation, when multiple attributes were set for the same key, in this case bug was causing that only last attribute was actually written to database and previous ones were skipped

### CODE REVIEW

**Review by yxieca — DISMISSED** [review]

File: syncd/Syncd.cpp [hunk 1]
```diff
@@ -2074,7 +2074,19 @@ void Syncd::syncUpdateRedisBulkQuadEvent(
 
         keys.push_back(key);
 
-        multiHash[key] = strAttributes.at(idx);
+        if (api == SAI_COMMON_API_BULK_SET)
+        {
+            // in case of bulk set operation, it can happen that multiple
+            // attributes will be set for the same key, then wehen we want to
```
Comment (lguohan) [inline syncd/Syncd.cpp:2080]: wehen -> when

**Comment by lguohan** [comment]
wonder if you add more description on the commit?

**Comment by kcudnik** [comment]
done

**Review by shi-su — APPROVED** [review]

**Comment by kcudnik** [comment]
retest vs please

**Comment by kcudnik** [comment]
@lguohan should all vs tests pass? in previous PR you mentioned that 4 tests are fine ?

**Comment by lguohan** [comment]
yes, if you see 4 tests failing, then it means this pr is ok. the four failed test cases are known issue.

**Review by yxieca — APPROVED** [review]

**Comment by lguohan** [comment]
retest this please

**Pull request merged**